src/Boot/src/Environment.php

```diff
@@ -51,6 +50,10 @@ public function getID(): string
      */
     public function set(string $name, $value): void
     {
+        if (isset($this->values[$name]) && !$this->overwrite) {
```

Comment:
```suggestion
        if (\array_key_exists($name, $this->values) && !$this->overwrite) {
```
It will work with predefined `null` value